CDP4JsonFileDal/JsonFileDal.cs

```diff
@@ -243,6 +248,8 @@ public override Task<IEnumerable<Thing>> Write(IEnumerable<OperationContainer> o
                     this.WriteModelReferenceDataLibraryToZipFile(modelReferenceDataLibraries, zipFile, path);
 
                     this.WriteIterationsToZipFile(iterations, zipFile, path);
+
+                    this.WriteExtraFilesToZipFile(files, zipFile, path);

```

Comment:
I would rename the "files" variable to reflect what it actually signifies: extensionFiles.

Reply:
agreed